src/utils.js

```diff
@@ -103,6 +108,28 @@ export function changeDateFormat(string) {
   return `${day}-${month}-${year}`
 }
 
+export function timeAgo(date) {
+  const seconds = Math.floor((new Date() - date) / 1000)
+  const years = Math.floor(seconds / 31536000)
+  const months = Math.floor(seconds / 2592000)
+  const days = Math.floor(seconds / 86400)
+  const hours = Math.floor(seconds / 3600)
+  const minutes = Math.floor(seconds / 60)
+
+  if (years > 1) {
+    return `${years} years ago`
+  } else if (months > 1) {
+    return `${months} months ago`
+  } else if (days > 1) {
+    return `${days} days ago`
+  } else if (hours > 1) {
+    return `${hours} hours ago`
+  } else if (minutes > 1) {
+    return `${minutes} minutes ago`
+  } else {
+    return `${Math.floor(seconds)} seconds ago`
```

Comment:
This has already been floored in line `112`.
```suggestion
    return `${seconds} seconds ago`
```